tools/tools_ml.py

```diff
@@ -27,3 +27,19 @@ def gexp_converter(data_entrez_list, cancer):
             except:
                 results.append('nan')
     return results
+
+def get_model_info(method, platform, cancer):
+    '''retrieves TMP model information
+    includes: model name, model parameters, and feature list (reported in TMP nomenclature)
+
+    specify which model you would like to look at. example: top performing jadbio gene expression only model
+
+    options method = ['skgrid', 'aklimate', 'cloudforest', 'subscope', 'jadbio']
+    options platform = ['GEXP', 'CNVR', 'MIR', 'MUTA', 'METH']
+    options cancer = ['BRCA', 'LGGGBM', 'COADREAD', 'SKCM', 'ACC', 'BLCA', 'CESC', 'ESCC', 'GEA', 'HNSC', 'KIRCKICH', 'KIRP', 'LIHCCHOL', 'LUAD', 'LUSC', 'MESO', 'OV', 'PAAD', 'PCPG', 'PRAD', 'SARC', 'TGCT', 'THCA', 'THYM', 'UCEC', 'UVM']
```

Comment:
ACC out of order

Reply:
updated the order of cancer cohorts in commit [9b09147](https://github.com/NCICCGPO/gdan-tmp-models/pull/54/commits/9b09147d35ceab6a336723b6c74ef06ac86c0ce2)